src/components/FileRepo/FileRepo.js

```diff
@@ -257,7 +258,9 @@ const FileRepo = compose(
 );
 
 export default props => (
-  <Gen3UserProvider
+  //Only has Gen3 integration, needs to be refactored to have all data repository integrations
```

Comment:
just wanna confirm we're ok with moving this forward without allowing user downloads :)

Reply:
yes this is acceptable. Apparently there wont be direct downloads from the DCF repository, so at this time this is ok.

There is new UI to support showing DCF files with a greyed out download button but this can be tackled in upcoming sprint. For now, file repo is only interested in Gen3 connection.

Reply:
got it, thanks! :)